lib/schemaUtils.js

```diff
@@ -2647,7 +2647,12 @@ module.exports = {
     }
 
     // handling authentication here (for http type only)
-    authHelper = this.getAuthHelper(openapi, operation.security);
+    if (options.alwaysInheritAuthentication) {
+      authHelper = this.getAuthHelper(openapi, openapi.security);
```

Comment:
@SahilChoudhary22 Inc ase when this is `true`, should we simply be using auth as `Inherit from paraent` rather then defining it same as parents? i.e. Value of authHelper should be `undefined` same as what's done in v2?

Reply:
@VShingala Correct! Asked the same [doubt](https://github.com/postmanlabs/openapi-to-postman/pull/451#discussion_r1143237537) from user on the parent PR. Missed it in this PR. Have fixed with the following commit - https://github.com/postmanlabs/openapi-to-postman/pull/728/commits/a942449c69f43d1c2ab9650f78c2643e0c76951c